scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -729,6 +759,104 @@ private void removeJobRuntimeData(Session session, long jobId) {
         session.getNamedQuery("deleteSelectorData").setParameter("jobId", jobId).executeUpdate();
     }
 
+    private void deleteInconsistentData(Session session) {
```

Comment:
How do we get that inconsistent data? It's good to clean up, but should it even exist?